examples/js/plugins/FeatureToolTip.js

```diff
@@ -84,17 +84,21 @@ var FeatureToolTip = (function _() {
             style = (geometry.properties && geometry.properties.style) || feature.style || layer.style;
             var context = { globals: {}, properties: getGeometryProperties(geometry) };
             style = style.drawingStylefromContext(context);
-            fill = style.fill.color;
-            stroke = '1.25px ' + style.stroke.color;
 
             if (feature.type === itowns.FEATURE_TYPES.POLYGON) {
                 symb = '&#9724';
+                fill = style.fill.color;
+                stroke = style.stroke && ('1.25px ' + style.stroke.color);
             } else if (feature.type === itowns.FEATURE_TYPES.LINE) {
                 symb = '&#9473';
                 fill = style.stroke.color;
```

Comment:
could you add check if `style` and `style.stroke` exits?

Reply:
added